doc/bash/loopback_aliases.md

```diff
@@ -0,0 +1,41 @@
+# Bash aliases for loopback #
+
+- [Loopback](#loopback)
+- [Loopback datasource](#loopback-datasource)
+- [Loopback model and property](#loopback-model-and-property)
+- [Loopback acl and relation](#loopback-acl-and-relation)
+- [Loopback remote-method](#loopback-remote-method)
+- [Loopback middleware](#loopback-middleware)
+- [Loopback boot-script](#loopback-boot-script)
+- [Loopback export-api-def](#loopback-export-api-def)
+- [Loopback swagger](#loopback-swagger)
+- [Back to main page](../../README.md)
+
+### Loopback ###
+- **lb**: `slc loopback`
+
+### Loopback datasouce ###
+- **lbd**: `slc loopback:datasouce`
+
+### Loopback model and property ###
+- **lbm**: `slc loopback:model`
+- **lbp**: `slc loopback:property`
+
+### Loopback acl and relation ###
+- **lba**: `slc loopback:acl`
+- **lbr**: `slc loopback:relation`
+
+### Loopback remote-method ###
+- **lbrm**: `slc loopback:remote-method`
+
+### Loopback middleware ###
+- **lbmd**: `slc loopback:middleware`
+
+### Loopback boot-script ###
+- **lbbr**: `slc loopback:boot-script`
```

Comment:
I am not able to find this alias in the aliases file. Have you forgot to add it? And by the way, shouldn't the alias be `lbbs` instead of `lbbr`?